Slam/DefaultAddMatchInteractor.swift

```diff
@@ -2,12 +2,20 @@ import Foundation
 
 public class DefaultAddMatchInteractor : AddMatchInteractor {
 
+    var api : SlamAPI?
```

Comment:
In general, it seems best to avoid optional types in the core of our app. I am attempting to limit my use of optionals to ViewControllers (and potentially Views).